serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1182,6 +1191,22 @@ export class _Exporter {
         }
     }
 
+    private setCameraTransformation(node: INode, babylonCamera: Camera, convertToRightHandedSystem: boolean): void {
+        if (!babylonCamera.position.equalsToFloats(0, 0, 0)) {
+            node.translation = convertToRightHandedSystem ? _GLTFUtilities._GetRightHandedPositionVector3(babylonCamera.position).asArray() : babylonCamera.position.asArray();
+        }
+
+        let rotation = babylonCamera.absoluteRotation;
+        let rotationQuaternion = Quaternion.RotationYawPitchRoll(rotation.y, rotation.x, rotation.z);

```

Comment:
Is this right? `absoluteRotation` is already a Quaternion.

Reply:
Good catch!